PrebidMobile.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+    <string>
+// Copyright 2018-2025 Prebid.org, Inc.
```

Comment:
I suggest using the `___YEAR___` macro here to avoid being strictly tied to the 2025 year.
Reference - https://help.apple.com/xcode/mac/9.0/index.html?localePath=en.lproj#/dev7fe737ce0